command/sub8_alarm/nodes/network_keepalive.py

```diff
@@ -18,5 +18,5 @@ def keepalive_pub(self, *args):
 
 if __name__ == '__main__':
     rospy.init_node('network_keepalive')
-    ka = KeepAlive(rospy.get_param('~auto'))
+    ka = KeepAlive(rospy.get_param('/autonomous'))
```

Comment:
In fact it might be better to do a check every time it publishes - that way we can go autonomous without restarting anything by just changing the param